fast64_internal/oot/c_writer/oot_room_writer/oot_room_shape_to_c.py

```diff
@@ -0,0 +1,96 @@
+from ....f3d.f3d_gbi import ScrollMethod, TextureExportSettings
+from ....utility import CData, PluginError
+from ...oot_utility import indent
+from ...oot_level_classes import OOTRoom, OOTRoomMeshGroup, OOTRoomMesh, OOTGfxFormatter
+
+
+def ootGetRoomShapeEntry(meshEntry: OOTRoomMeshGroup, roomShape: str):
+    """Returns a single room shape entry"""
+    opaqueName = meshEntry.DLGroup.opaque.name if meshEntry.DLGroup.opaque is not None else "NULL"
+    transparentName = meshEntry.DLGroup.transparent.name if meshEntry.DLGroup.transparent is not None else "NULL"
+    roomShapeEntry = "{ "
+
+    if roomShape == "ROOM_SHAPE_TYPE_CULLABLE":
+        roomShapeEntry += (
+            "{ "
+            + ", ".join([f"{pos}" for pos in meshEntry.cullGroup.position])
+            + "}, "
+            + f"{meshEntry.cullGroup.cullDepth}, "
+        )
+    elif roomShape == "ROOM_SHAPE_TYPE_IMAGE":
+        raise PluginError("Pre-Rendered rooms not supported.")
+
+    roomShapeEntry += f"{opaqueName}, {transparentName}" + " }\n"
```

Comment:
```suggestion
    roomShapeEntry += f"{opaqueName}, {transparentName}" + " }"
```

Reply:
since this is an array I think it should be ``" },\n"``, though I don't think it's ever exporting multiple values here (also that ``\n`` is to get the closing brackets of the array on the next line)

Reply:
nvm I missed the other comment lol